src/app/components/carousel/carousel.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="section team-carousel" id="carousel">
+    <div class="container">
+        <div class="tim-title">
+            <h3>Our Team</h3>
```

Comment:
Shall we use <h2> and the CSS class "title"? So it will look like the above Team Members section.